code/krepel/log.d

```diff
@@ -59,44 +67,42 @@ template LogMessageDispatch(LogLevel Level)
 
 alias Info    = LogMessageDispatch!(LogLevel.Info);
 alias Warning = LogMessageDispatch!(LogLevel.Warning);
-alias Error   = LogMessageDispatch!(LogLevel.Error);
-
+alias Failure = LogMessageDispatch!(LogLevel.Failure);
 
 void StdoutLogSink(LogLevel Level, char[] Message)
 {
+  static import std.stdio;
+
   final switch(Level)
   {
-    case LogLevel.Info:    io.write("Info: ");    break;
-    case LogLevel.Warning: io.write("Warning: "); break;
-    case LogLevel.Error:   io.write("Error: ");   break;
+    case LogLevel.Info:    std.stdio.write("Info: "); break;
+    case LogLevel.Warning: std.stdio.write("Warn: "); break;
+    case LogLevel.Failure: std.stdio.write("Fail: "); break;
   }
```

Comment:
Btw I'm using "Warn" and "Fail" instead of "Warning" and "Failure" so the output is aligned on the console:

```
Warn: Something might be wrong...
Info: Finished stuff.
Fail: Well, you could've tried harder, you know?
```